src/Aspire.Hosting.Azure.EventHubs/AzureEventHubsExtensions.cs

```diff
@@ -322,6 +322,15 @@ public static IResourceBuilder<AzureEventHubsResource> RunAsEmulator(this IResou
                 // Deterministic file path for the configuration file based on its content
                 var configJsonPath = aspireStore.GetFileNameWithContent($"{builder.Resource.Name}-Config.json", tempConfigFile);
 
+                // The docker container runs as a non-root user, so we need to grant other user's read/write permission
+                if (!OperatingSystem.IsWindows())
+                {
+                    var mode = UnixFileMode.UserRead | UnixFileMode.UserWrite | UnixFileMode.UserExecute |
+                               UnixFileMode.OtherRead | UnixFileMode.OtherWrite | UnixFileMode.OtherExecute;
```

Comment:
```suggestion
                    var mode = UnixFileMode.UserRead | UnixFileMode.UserWrite |
                               UnixFileMode.GroupRead |
                               UnixFileMode.OtherRead;
```

The file doesn't need to be executed by anyone. And it shouldn't be written by anyone else. We should also add GroupRead to make it coherent.